ansible/ncn-sysctl.yml

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ansible-playbook
+#
+# MIT License
+#
+# (C) Copyright 2022-2023 Hewlett Packard Enterprise Development LP
```

Comment:
If this is a new file, should only have the current year in the copyright header, I think.

```suggestion
# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
```

Reply:
Updated